### PR TITLE
ft: item delete modal

### PR DESCRIPTION
**Pull request by DimejiAre**
## Description
Describe change or new feature here.
## Type of change
Please delete options that are not relevant.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update
## Checklist
Remove any items which are not applicable.
- [x] I have performed a self-review of my own code
- [x] My code follows the style guidelines of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works .
- [ ] New and existing unit tests pass locally with my changes
## Link to Trello card
https://trello.com/c/b8BmH7EV

### CODE REVIEW

**Comment by DimejiAre** [comment]
[item delete modal](https://trello.com/c/b8BmH7EV/52-item-delete-modal)

File: src/components/inventory/expand.js [hunk 1]
```diff
@@ -1,9 +1,34 @@
...
+    confirm({
+      title: 'Are you sure you want to delete this item?',
+      okText: 'Yes',
+      okType: 'danger',
+      cancelText: 'No',
+      onOk () {
+        AxiosAuth().delete(`https://shopping-cart-eu3-staging.herokuapp.com/api/store/products/${id}`)
+          .then(res => {
+            dispatch(creators.getCurrentUser())
+            message.success('item deleted')
```
Comment (shaunorpen) [inline src/components/inventory/expand.js]: Assuming this message is displayed to the user, can we capitalise it?

**Review by shaunorpen — CHANGES_REQUESTED** [review]
Minor tweak then I'm happy :).

File: src/components/inventory/expand.js (hunk 1 above)
Comment (DimejiAre) [reply src/components/inventory/expand.js]: @shaunorpen I'll capitalize first letter of each word, to remain consistent with rest of the codebase. Is that okay? or you think we should change everything to all caps?

Comment (shaunorpen) [reply src/components/inventory/expand.js]: Just capitalise to maintain consistency I think. Cheers!

Comment (DimejiAre) [reply src/components/inventory/expand.js]: @shaunorpen done

Comment (kip-guile) [reply src/components/inventory/expand.js]: Looks Good

**Review by kip-guile — APPROVED** [review]
Everything looks good. Should work

**Pull request merged**